packages/core/base/src/services/crossmintStatusService.ts

```diff
@@ -1,6 +1,6 @@
-import { clientNames, customHeaders, onboardingRequestStatusResponse, OnboardingQueryParams } from "../models/types";
-import { validate } from "uuid";
+import { OnboardingQueryParams, clientNames, customHeaders, onboardingRequestStatusResponse } from "../models/types";
 import { getEnvironmentBaseUrl } from "../utils/ui";
+import validateUUID from "../utils/validateUUID";
```

Comment:
This is actually the very same logics that `uuid` package uses 😄 

See https://github.com/uuidjs/uuid/blob/main/src/validate.js